wrappers/java/pom.xml

```diff
@@ -25,7 +25,7 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>2.9.9.1</version>
+      <version>[2.9,2.10)</version>
```

Comment:
Seeing as we are already modifying this, this should be [2.9.9.2,2.10) - just to be sure someone doesn't _e.g._ accidentally use a previously installed vulnerable version.

Reply:
```suggestion
       <version>[2.9.9.2,2.10)</version> 
```